test/fixtures/env.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   BEEP: false,
   BOOP: [ 'some', 'thing', 'that', 'goes', 'wow' ],
   BLEEP: 'false',
-  PING: [ 'ping', 'true', 2, 100 ]
+  PING: [ 'ping', 'true', 2, 100 ],
+  PONG: 'some,thing,that,goes,wow'
```

Comment:
@rodw1995 Can you please adjust this line so that it is encapsulated in quotes?  Can you also add documentation to Readme.md file that describes how you can wrap a line in quotes?

```diff
-PONG: 'some,thing,that,goes,wow'
+PONG: '"some,thing,that,goes,wow"'
```